docs/ProductionChecklistForScalarDBCluster.md

```diff
@@ -0,0 +1,73 @@
+# Production checklist for ScalarDB Cluster
+
+This checklist provides recommendations when deploying ScalarDB Cluster in a production environment.
+
+## Before you begin
+
+In this checklist, we assume that you are deploying ScalarDB Cluster on a managed Kubernetes cluster, which is recommended.
+
+## Production checklist: ScalarDB Cluster
+
+The following is a checklist of recommendations when setting up ScalarDB Cluster in a production environment.
+
+### Number of pods and Kubernetes worker nodes
+
+To ensure that the Kubernetes cluster has high availability, you should use at least three worker nodes in three availability zones and deploy at least three pods. You also should ensure that one pod is deployed on one worker node by using [Node Affinity](https://kubernetes.io/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity/).
```

Comment:
Can it be achieved as long as users use our helm charts and use three worker nodes?
If so, I think it's helpful to mention it.

Reply:
Users can achieve it by using Scalar Helm Chart with a specific configuration.
We provide the sample configuration for this in the following sample.
https://github.com/scalar-labs/scalar-kubernetes/blob/3fb947ebd24be61fbe58b157e2abb19fdd422a1f/conf/scalardb-cluster-custom-values-direct-kubernetes-mode.yaml#L65-L86

So, I will add the link to this sample configuration.

Reply:
I added the link for the sample configuration in 4b5b5176989a33f5638f5892134528be21152f2d.
Note: This sample configuration file is under review in another PR. So, at this time this link returns 404. You can see the actual sample in the following URL.
https://github.com/scalar-labs/scalar-kubernetes/blob/3fb947ebd24be61fbe58b157e2abb19fdd422a1f/conf/scalardb-cluster-custom-values-direct-kubernetes-mode.yaml#L65-L86